cube/char_set.cpp

```diff
@@ -1,5 +1,5 @@
 /**********************************************************************
- * File:        char_samp_enum.cpp
```

Comment:
I added these old file names just for uniformity's sake to be consistent with the other files.